docs/Using Fleet/Understanding-host-vitals.md

```diff
@@ -713,15 +692,6 @@ SELECT
   '' AS vendor,
   path AS installed_path
 FROM chocolatey_packages
-UNION
-SELECT
-  name AS name,
-  version AS version,
-  'Package (Atom)' AS type,
-  'atom_packages' AS source,
-  '' AS vendor,
-  path AS installed_path
-FROM cached_users CROSS JOIN atom_packages USING (uid);
 ```
```

Comment:
Did you generate these with `make generate-doc`? (Just in case.)

Reply:
```
sharonkatz@Sharons-MacBook-Pro fleet % make generate-doc
mkdir -p build/linux
mkdir -p build/darwin
go generate github.com/fleetdm/fleet/v4/server/fleet
go generate github.com/fleetdm/fleet/v4/server/service/osquery_utils
sharonkatz@Sharons-MacBook-Pro fleet %
```

Reply:
@lucasmrod running `make generate-doc` has no effect on this file. TMWYT